ontology/yaml/resources/ELECTRICAL/entity_types/UPS.yaml

```diff
@@ -74,3 +74,11 @@ UPS_PWM:
   implements:
   - UPS
   - PWM
+
+UPS_BBM_UPSM:
```

Comment:
Based on above comments, should this instead be `UPS_SS_UPSBM`

Reply:
Thanks for the feedback on this, really cleared up a lot of concerns I had. I've updated the PR based on your comments.